diff --git a/node_modules/@types/chec__commerce.js/features/checkout.d.ts b/node_modules/@types/chec__commerce.js/features/checkout.d.ts
index 88c111f..e911709 100755
--- a/node_modules/@types/chec__commerce.js/features/checkout.d.ts
+++ b/node_modules/@types/chec__commerce.js/features/checkout.d.ts
@@ -115,7 +115,7 @@ export class Checkout {
         token: string,
         data: { shipping_option_id: string; country: string; region?: string | undefined },
     ): Promise<CheckShippingOptionResponse>;
-    getShippingOptions(token: string, data: { country: string; region?: string | undefined }): Promise<GetShippingOptionsResponse>;
+    getShippingOptions(token: string, data: { country: string; region?: string | undefined }): Promise<GetShippingOptionsResponse[]>; // I changed this with patch-package from Promise<GetShippingOptionsResponse> to Promise<GetShippingOptionsResponse[]>
     setTaxZone(
         token: string,
         data: { ip_address?: string | undefined; country?: string | undefined; region?: string | undefined; postal_zip_code?: string | undefined },
diff --git a/node_modules/@types/chec__commerce.js/types/category.d.ts b/node_modules/@types/chec__commerce.js/types/category.d.ts
index baf3336..5bed953 100755
--- a/node_modules/@types/chec__commerce.js/types/category.d.ts
+++ b/node_modules/@types/chec__commerce.js/types/category.d.ts
@@ -1,3 +1,5 @@
+import { Asset } from "@chec/commerce.js/types/asset";
+
 export interface Category {
     id: string;
     slug: string;
@@ -6,4 +8,5 @@ export interface Category {
     products: number;
     created: number;
     meta: any;
+    assets: Asset[] // I added with patch-package
 }
diff --git a/node_modules/@types/chec__commerce.js/types/line-item.d.ts b/node_modules/@types/chec__commerce.js/types/line-item.d.ts
index 28d2908..ce1a51d 100755
--- a/node_modules/@types/chec__commerce.js/types/line-item.d.ts
+++ b/node_modules/@types/chec__commerce.js/types/line-item.d.ts
@@ -1,6 +1,8 @@
 import { Price } from './price';
 import { SelectedVariant } from './selected-variant';
 
+// edited with patch-package
+
 export interface LineItem {
     id: string;
     name: string;
@@ -8,8 +10,22 @@ export interface LineItem {
     product_id: string;
     product_name: string;
     sku: string;
-    media: any; // todo
-    variants: SelectedVariant[];
+    // media: any; // todo // edited by me
+    // variants: SelectedVariant[]; // edited by me
     price: Price;
     line_total: Price;
+    // added by me ---
+    is_valid: boolean;
+    selected_options: {
+      group_id: string;
+      group_name: string;
+      option_id: string;
+      option_name: string;
+      price: Price;
+    }[];
+    permalink: string;
+    media: { source: string; type: string };
+    product_meta: any[];
+    variant: any;
+    // --- end added by me
 }
